cdap-watchdog/src/main/java/co/cask/cdap/metrics/collect/LocalMetricsCollectionService.java

```diff
@@ -129,22 +128,24 @@ protected void shutDown() throws Exception {
 
   /**
    * Creates a task for cleanup.
-   * @param retention Retention in seconds.
+   *
+   * @param secondRetention Retention in seconds for second resolution table.
    */
-  private Runnable createCleanupTask(final long retention) {
+  private Runnable createCleanupTask(long secondRetention) {
```

Comment:
Seems like this is only used for scheduling. Better call this `nextScheduleDelaySecs` and update the javadoc accordingly.